### ISSUE TITLE
Flatscreen UI.Input Tab Key

### ISSUE THREAD

**Issue by briantcluff** [StereoKit/StereoKit#524]
## Description
Currently in the flatscreen simulator if you have UI.Input with focus and you hit "TAB" it inserts a tab character into the input text instead of traversing to the next focusable form element. I don't know if traditional platform specific XR keyboards even have the tab key, but the behavior was unexpected.

Again, this is probably the lowest priority issue of all the wonderful things you are working on. Just thought it might be worth documenting.

## Platform / Environment
flatscreen simulator

## Logs or exception details
N/A

**Comment by maluoi** [comment]
Yeah, thanks for bringing it up! StereoKit doesn't currently support tab navigation or general keyboard focus, so _for the moment_ inserting a tab character is probably the correct behavior.

I do have plans to revise this in upcoming updates, there's plenty of accessibility work that still needs to go into the UI! Not to mention figuring out what exactly tab order might mean in a 3D environment :D
